docs/GettingStarted.rst

```diff
@@ -1145,6 +1148,47 @@ the link.
 
     .. note:: If the activity composer is empty, the |image78| button still grey and unclickable until adding something (a text message, a link or a file) in the activity composer area.
 
+.. _Sharing-news:
+
+	
+Sharing a news in the activity stream
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	
+From a space's acivity stream, you are able to share a news with the space’s members. 
+Publishing a news allows you to easily write, broadcast, pin and share communication content.
+
+
+Sharing a simple news from short form
+--------------------------------------
+
+To share an article with other space’s members, click on News tab from the activity composer, the tab contains :
+
+
+- A **Title** field :  Allows you to enter the news title. The title should not exceed 150 characters: Beyond that limit you will not be allowed to write.
+
+- A **Content** field : Allows you to enter the content of the News. No limitation for the number of characters.
+
+- A **Pin article** Check box.
+
+- A **More icon** with a tooltip “more options”: Permits to open the creation full form.
+
+- A **Post** button : Disabled by default until the two fields “Title” and “Content” are filled.
+
+|image156|
+
+
+Then type your News Title and content in the appropriate fields and click on Post button
```

Comment:
There is a "then" here but no first step, it's weird. I would replace this sentence by "Once all fields filled, click on the Post button to post the News in the space activity stream." and remove the next sentence which is useless IMO. WDYT ?